src/main/java/de/ait/secondlife/services/OfferServiceImpl.java

```diff
@@ -78,7 +85,6 @@ public OfferResponseDto createOffer(OfferCreationDto dto) throws CredentialExcep
             Offer newOffer = mappingService.toEntity(dto);
             newOffer.setUser(user);
             newOffer.setCategory(categoryService.getCategoryById(dto.getCategoryId()));
-            newOffer.setLocation(locationService.getLocationById(dto.getLocationId()));
```

Comment:
why are we deleting this location setter?

Reply:
fixed.